### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 31

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#207] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254561-08ebfa76-b8d0-4027-9a0f-9839268cd9b6.png)

**Comment by LuginiAndrea** [comment]
Problema simile a quello delle targhe.
1) $$26^3 \cdot 10^3 \cdot 1$$
2) $$26^3 \cdot 10^3 + 26^2 \cdot 10^3 \cdot 3 - 26^2 \cdot 10^3$$
3) $$\left(26^2 \cdot 3\right) \cdot \left(10^2  \cdot 3\right) \cdot 3$$

**Comment by niccolo-fato** [comment]
@LuginiAndrea la terza è sbagliata: (25^2 ⋅ 3) ⋅ (9^2 ⋅ 3) ⋅ 3
